From 639c53bc8ea7fb2cdcedb2801c65c0e24c014291 Mon Sep 17 00:00:00 2001
From: Mirza Krak <mirza.krak@northern.tech>
Date: Thu, 23 Aug 2018 13:45:48 +0200
Subject: [PATCH 3/3] thirdparty: date: remove usage of SYSTEM

The SYSTEM option hints that include paths are meant
as system includes (-isystem).

This produces errors like:

cstdlib:75:15: fatal error: stdlib.h: No such file or directory

In Yocto

Signed-off-by: Mirza Krak <mirza.krak@northern.tech>
---
 thirdparty/date/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/thirdparty/date/CMakeLists.txt b/thirdparty/date/CMakeLists.txt
index 683ddce..7f836ca 100644
--- a/thirdparty/date/CMakeLists.txt
+++ b/thirdparty/date/CMakeLists.txt
@@ -57,7 +57,7 @@ else( )
 	target_compile_definitions( tz PRIVATE -DHAS_REMOTE_API=1 )
 	target_compile_definitions( tz PUBLIC -DUSE_OS_TZDB=0 )
 	find_package( CURL REQUIRED )
-	include_directories( SYSTEM ${CURL_INCLUDE_DIRS} )
+	include_directories( ${CURL_INCLUDE_DIRS} )
 	set( OPTIONAL_LIBRARIES ${CURL_LIBRARIES} )
 endif( )
 
-- 
2.18.0

